tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,73 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: upload a file with virus to a user share
```

Comment:
```suggestion
  Scenario Outline: upload a virus file to a user share
```

Reply:
I think "file with virus" and "virus file" same